File: src/main/Java/boj_step2/boj2480.java
import java.io.*;
import java.util.StringTokenizer;

public class boj2480 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        int diceA = Integer.parseInt(tokenizer.nextToken());
        int diceB = Integer.parseInt(tokenizer.nextToken());
        int diceC = Integer.parseInt(tokenizer.nextToken());
        if (diceA == diceB && diceA == diceC) writer.append(String.valueOf(10_000 + diceA * 1000));
        else if (diceA == diceB || diceA == diceC) writer.append(String.valueOf(1000 + diceA * 100));
        else if (diceB == diceC) writer.append(String.valueOf(1000 + diceB * 100));
        else { writer.append(String.valueOf(Math.max(diceA, Math.max(diceB, diceC)) * 100)); }
        writer.close();
    }
}

File: src/main/kotlin/others/boj2480.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import kotlin.math.max

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val (a, b, c) = r.readLine().split(" ").map { it.toInt() }
    if(a == b && a == c) w.write((10000 + a * 1000).toString())
    else if(a == b || a == c) w.write((1000 + a * 100).toString())
    else if(b == c) w.write((1000 + b * 100).toString())
    else w.write((max(max(a, b), c) * 100).toString())
    w.close()
}